src/components/ChainVerificationAnimation/Phase.tsx

```diff
@@ -31,6 +31,9 @@ export const PhaseColumn: React.FC<PhaseProps> = ({
   const isCompleted = status === VerificationStatus.VERIFIED;
   const showTick = isCompleted;
 
+  // Calculate animation duration based on number of steps. Ensure minimum duration of 1 second even if no steps
+  const animationDuration = steps.length > 0 ? `${steps.length}s` : "1s";
```

Comment:
A more idiomatic way to do this would be:

```suggestion
  const animationDuration = `${Math.min(1, steps.length)}s`;
```

Reply:
Gus, if we use Math.min(1, steps.length), it doesn't work. The explanation from Sonnet is:

If steps.length is 0: Math.min(1, 0) → 0 second (animation wouldn't show)
If steps.length is 3: Math.min(1, 3) → 1 second (caps at 1 second)
If steps.length is 5: Math.min(1, 5) → 1 second (caps at 1 second)

This would limit the animation to a maximum of 1 second regardless of how many steps there are, which defeats the purpose of dynamically scaling the animation duration based on the number of steps.

What we need is Math.max(1, steps.length) to ensure:

The minimum duration is 1 second (for empty or single-step phases)
Longer phases get proportionally longer animations

Reply:
Sorry, yes, I meant `max`.